hathor/transaction/base_transaction.py

```diff
@@ -1185,7 +1185,7 @@ def __repr__(self) -> str:
 
     def __str__(self) -> str:
         cls_name = type(self).__name__
-        value_str = hex(self.value) if self.is_token_authority else str(self.value)
+        value_str = hex(self.value) if self.is_token_authority() else str(self.value)
```

Comment:
Was it a bug?

Reply:
Yes, it looks like it. Previous versions of mypy didn't catch it. I guess it wasn't a problem because this is the `__str__` method, so probably only used for logging

Reply:
Got it. Thanks!